lib/node_modules/@stdlib/string/next-grapheme-cluster-break/lib/main.js

```diff
@@ -65,13 +69,20 @@ function nextBreak( str, idx ) {
 	var mid;
 	var i;
 
+	if ( !isString( str ) ) {
+		throw new TypeError( 'invalid argument. Must provide a string. Value: `' + str + '`.' );
```

Comment:
```suggestion
		throw new TypeError( 'invalid argument. First argument must be a string. Value: `' + str + '`.' );
```

As there are multiple arguments, we need to be a bit more specific.